### PR TITLE
LUCENE-9954: README for Luke

### PR DESCRIPTION
**Pull request by michaelwechner**
A README for Luke at https://github.com/apache/lucene/tree/main/lucene/luke whereas see https://issues.apache.org/jira/browse/LUCENE-9954

### CODE REVIEW

File: lucene/luke/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    limitations under the License.
+ -->
+
+# Luke
+
+![Luke, the Lucene Toolbox Project](luke-logo.gif) + Swing ![Java](java_logo.png)
+
+## Introduction
+
+Luke (atop Swing) is the GUI tool for introspecting your Lucene / Solr / Elasticsearch index. It allows:
```
Comment (mocobeta) [inline lucene/luke/README.md:24]:
```suggestion
Luke (atop Swing) is the GUI tool for introspecting your Lucene index. It allows:
```
Maybe we should mention only Lucene here.

File: lucene/luke/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Launching Luke
+
+Using a binary release
+
+The [Lucene Downloads](https://lucene.apache.org/core/downloads.html) page contains stable releases of Lucene including Luke. Download the respective release, unpack and run a shell or batch scripts inside the folder 'luke' (or directly with java -jar if you prefer it that way).
+
+## Older releases
+
+Older releases of Luke can be found at https://github.com/DmitryKey/luke
```
Comment (mocobeta) [inline lucene/luke/README.md:41]:
```suggestion
Older releases of Luke (prior to 8.1) can be found at https://github.com/DmitryKey/luke
```
We can be more specific about the "older version"?

File: lucene/luke/README.md (hunk 1 above)
Comment (michaelwechner) [reply lucene/luke/README.md:24]: sounds good to me as well

File: lucene/luke/README.md (hunk 2 above)
Comment (michaelwechner) [reply lucene/luke/README.md:41]: sounds good to me as well

File: lucene/luke/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ -->
+
+# Luke
+
+![Luke, the Lucene Toolbox Project](luke-logo.gif) + Swing ![Java](java_logo.png)
```
Comment (mocobeta) [inline lucene/luke/README.md:20]: Though I love the old Luke logo, recently Lucene logo (the origin of it) was replaced with a new one.
https://lucene.apache.org/

I think it'd be better to remove the old-style Logo for now (until we get a new logo for Luke :)

Comment (mocobeta) [reply lucene/luke/README.md:20]: Yes, let's remove this line and delete the all unused files.

Comment (michaelwechner) [reply lucene/luke/README.md]: Done :-) see https://github.com/wyona/lucene/tree/LUCENE-9954_README4Luke/lucene/luke

**Review by mocobeta — APPROVED** [review]
It looks good to me. Thanks @michaelwechner

**Pull request merged**